bedrock/base/templates/includes/global-nav.html

```diff
@@ -211,6 +211,11 @@ <h3 class="summary" data-id="technology">
                   </a>
                 </li>
                 <li>
+                  <a href="{{ url('mozorg.developer') }}" data-link-type="nav" data-link-name="Mozilla Developers" data-link-group="Technology">
+                    {{ _('Mozilla Developers') }}
```

Comment:
I added a subnav to the technology page and it doesn't look too bad. I don't like that one link is "Technology" and another is "Web Innovations" but both lead to the same page... that is perhaps a battle for another day.